templates/releases/feed.xml

```diff
@@ -23,7 +23,7 @@
         <entry>
```

Comment:
A couple lines above you need `            {%- set link = "/crate/" ~ name ~ "/" ~ version -%}`, not `release.name`. Same for both branches of the `if`.